tests/sentry/eventstore/snuba/test_backend.py

```diff
@@ -116,9 +126,36 @@ def test_get_next_prev_event_id(self):
         assert self.eventstore.get_prev_event_id(None, filter=filter) is None
         assert self.eventstore.get_next_event_id(None, filter=filter) is None
 
-    def test_get_transaction_event_by_id(self):
+    def test_transaction_get_event_by_id(self):
         event = self.eventstore.get_event_by_id(self.project2.id, self.transaction_event.event_id)
 
         assert event.id == "d" * 32
         assert event.get_event_type() == "transaction"
         assert event.project_id == self.project2.id
+
+    def test_transaction_get_next_prev_event_id(self):
+
+        filter = Filter(
+            project_ids=[self.project1.id, self.project2.id],
+            conditions=[["type", "=", "transaction"]],
+        )
+
+        event = self.eventstore.get_event_by_id(self.project2.id, "e" * 32)
+
+        prev_event = self.eventstore.get_prev_event_id(event, filter=filter)
+
+        next_event = self.eventstore.get_next_event_id(event, filter=filter)
+
+        assert prev_event == (six.text_type(self.project2.id), "d" * 32)
+
+        assert next_event is None
+
+        event = self.eventstore.get_event_by_id(self.project2.id, "d" * 32)
+
+        prev_event = self.eventstore.get_prev_event_id(event, filter=filter)
+
+        next_event = self.eventstore.get_next_event_id(event, filter=filter)
+
+        assert prev_event is None
+
+        assert next_event == (six.text_type(self.project2.id), "e" * 32)
```

Comment:
you may wantto pack these lines a bit more. Separating them with an empty line each time does not help because you are not grouping them in logically related blocks